webapp/src/components/sidebar_right/index.tsx

```diff
@@ -99,29 +82,15 @@ function mapPrsToDetails(prs: Item[], details: Item[]) {
         return {
             ...pr,
             status: foundDetails.status,
-            approvers: foundDetails.approvers,
+            num_approvers: foundDetails.num_approvers,
             total_reviewers: pr.reviewers.length,
         };
     });
 }
 
 function SidebarRight({theme}: {theme: Theme}) {
-    const {username, yourAssignments, org, unreads, gitlabURL, rhsState, reviews, yourPrs, reviewDetails, yourPrDetails, getUserLocale} = useSelector((state: GlobalState) => {
-        const store = getPluginState(state);
-        return {
-            username: store.username,
-            reviews: store.reviews,
-            reviewDetails: store.reviewDetails,
-            yourPrs: store.yourPrs,
-            yourPrDetails: store.yourPrDetails,
-            yourAssignments: store.yourAssignments,
-            unreads: store.unreads,
-            org: store.organization,
-            gitlabURL: store.gitlabURL,
-            rhsState: store.rhsState,
-            getUserLocale: getCurrentUserLocale(state),
-        };
-    });
+    const sidebarData = getSidebarData(useSelector((state) => state));
```

Comment:
```suggestion
    const sidebarData = useSelector(getSidebarData);
```
I think it should be like this.